modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/general.jsp

```diff
@@ -42,6 +42,17 @@ portletDisplay.setURLBackTitle(ParamUtil.getString(request, "backURLTitle"));
 	onSubmit="event.preventDefault();"
 	wrappedFormContent="<%= false %>"
 >
+	<aui:script sandbox="<%= true %>">
+		var groupElement = document.getElementById('<portlet:namespace />groupLabel');
+
+		if (groupElement) {
+			groupElement.setAttribute(
+				'aria-label',
+				'<%= LanguageUtil.get(request, "general") %>'
+			);
+		}
+	</aui:script>
```

Comment:
This look like it will result in flash of old label when the page is opened. This is probably fine, I don't think `aria-label` needs to be available immediately. Is it safe to leave it like this?

Reply:
It should be safe as you said it will not be immediately available to the Screen Reader.